runner.rb

```diff
@@ -13,6 +14,8 @@
 
   computer = Computer.new(board)
 
+  win_checker = WinChecker.new(board, player1, computer)
```

Comment:
Creating a new class here was a great way to consolidate our code! I also like how you implemented the draw scenario with the each method.